src/Tizen.NUI/src/internal/ItemView.cs

```diff
@@ -193,8 +193,8 @@ public void RemoveLayout(uint layoutIndex)
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
-        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("Deprecated in API9, will be removed in API11. Please use ChildCount property instead!")]
```

Comment:
We can delete Hidden API(API with EditorBrwsable) without ACR process.
Why don't you delete the API right away?